src/components/DpLabel/DpLabel.vue

```diff
@@ -23,15 +23,16 @@
 </template>
 
 <script>
-import { CleanHtml } from '../../directives/CleanHtml/CleanHtml'
+import { CleanHtml, Tooltip } from '../../directives'
 import { de } from '../shared/translations'
 import { prefixClassMixin } from '@demos-europe/demosplan-utils'
 
 export default {
   name: 'DpLabel',
 
   directives: {
-    cleanhtml: CleanHtml
+    cleanhtml: CleanHtml,
+    tootip: Tooltip
```

Comment:
The approve was premature, there's a typo :see_no_evil: (here and other occurrences)
```suggestion
    tooltip: Tooltip
```

Reply:
luckily its still a draft ;-)

Reply:
fixed in 54060a05b7048f10d09efe3170dcb3ed76f0911a